fuseutil/file_system.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"github.com/jacobsa/fuse"
 	"github.com/jacobsa/fuse/fuseops"
+	"github.com/panjf2000/ants/v2"
```

Comment:
Not keen on adding a new dependency just for this change. If we decide to go forward with this change, we should implement it without extra dependencies. Check out https://github.com/sourcegraph/gophercon-2018-liveblog/issues/35 for inspiration